src/Provider/Provider.Web/Utility.cs

```diff
@@ -75,11 +75,7 @@ public static IList<string> GetValidRoutesForVacancy(Vacancy vacancy)
             if (string.IsNullOrWhiteSpace(vacancy.NumberOfPositions?.ToString()))
                 return validRoutes;
 
-            validRoutes.AddRange(new[] { RouteNames.ShortDescription_Post, RouteNames.ShortDescription_Get });
-            if (string.IsNullOrWhiteSpace(vacancy.ShortDescription))
-                return validRoutes;
-
-            validRoutes.AddRange(new[] 
+           validRoutes.AddRange(new[] 
```

Comment:
You'll need to add `ShortDesciprtion` this to `VacancyHasStartedPartTwo` and update tests